### PR TITLE
Updates default TGUI background, overhauls autolathe tgui theme

### PR DESCRIPTION
**Pull request by thgvr**
## About The Pull Request
Adds shiptest SVG as a replacement for the NT one in default tgui theme
Overhauls `ntos_terminal` theme, giving it an orange hue and adding it to autolathes.
image
 

## Changelog

:cl:
add: Autolathe has a new TGUI theme.
imageadd: New TGUI background, the Shiptest logo.
/:cl:

### CODE REVIEW

File: tgui/packages/tgui/interfaces/Autolathe.js [hunk 1]
```diff
@@ -220,7 +220,7 @@ export const Autolathe = (props, context) => {
                           maxValue={design.maxmult}
                           disabled={design.buildable}
                           backgroundColor={
-                            design.buildable ? '#999999' : 'default'
+                            design.buildable ? '#0000000' : 'default'
```
Comment (MarkSuckerberg) [inline tgui/packages/tgui/interfaces/Autolathe.js]:
```suggestion
                            design.buildable ? '#00000000' : 'default'
```

**Review by MarkSuckerberg — CHANGES_REQUESTED** [review]
pog

**Comment by thgvr** [comment]
Not sure if I'm sold on the orange, might take it back to green. If anyone has thoughts pls comment

**Comment by thgvr** [comment]
@MarkSuckerberg what do you think of the orange vs green

**Comment by MarkSuckerberg** [comment]
> @MarkSuckerberg what do you think of the orange vs green

I really do like the amber, I'd say increase the contrast on the text a bit though if you could. 

honestly the classic amber display would be amber text on a black background, even if that's a bit less colourful and distinct from normal UIs

**Comment by thgvr** [comment]
im gonna make it green

**Review by MarkSuckerberg — APPROVED** [review]

**Pull request merged**